src/aspire/apple/apple.py

```diff
@@ -113,10 +113,9 @@ def process_micrograph(self, filepath, return_centers=True, return_img=False, sh
         particle_image = None
         if create_jpg and self.output_dir is not None:
             particle_image = self.particle_image(picker.original_im, picker.particle_size, centers)
-            misc.imsave(
-                os.path.join(self.output_dir, os.path.splitext(os.path.basename(picker.filename))[0] + '_result.jpg'),
-                particle_image
-            )
+            image_tmp = Image.fromarray((particle_image).astype(np.uint8))
+            image_tmp.save(os.path.join(self.output_dir, os.path.splitext(
```

Comment:
Let's avoid oneliners when we can. Separate the filename creation into multiple steps.

Reply:
Done in new commit 7216539.